### PR TITLE
Adding SOCK

### PR DESCRIPTION
**Pull request by mruizh**
Adding SOCK

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -4043,5 +4043,19 @@
...
+  "ticker": "SOCK",
+   "poolId": "564e6152a892f02d3456bf3a5e00ad0aa3806369d1b152a8abae435b",
+   "social": {
+     "website": "https://sockstake.io",
+     "twitter": "ADAPoolSock",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+   },
+   "telegram": "SockStakeAdaPool",
```
Comment (Bez625) [inline registry.json:4057]: Please change this to be the handle of an individual and not a group or channel

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 1 above)
Comment (mruizh) [reply registry.json:4057]: Updated, blank now.

Comment (Bez625) [reply registry.json:4057]: Please can you change this to be the handle of an individual and not a group or channel? @sockstakeadapool is a channel...

**Review by Bez625 — CHANGES_REQUESTED** [review]

**Comment by Bez625** [comment]
Please address the comments or we will close in the coming weeks

**Comment by Bez625** [comment]
Closing this as there has been no activity. Please raise another PR if you still wish to be included in the alliance.

**Pull request closed without merging**